scripts/population/README.md

```diff
@@ -0,0 +1,9 @@
+
+## Importing country population data
+
```

Comment:
Document here where the files  should be located and how it should be named.
It seems that the .tgz extension is hardcoded. Which is understandable, but it should be exposed beforehand here.